src/tasks/leveling.ts

```diff
@@ -754,33 +878,37 @@ export const LevelingQuest: Quest = {
       limit: { tries: 12 },
     },
     {
-      name: "Use Ointment of the Occult",
+      name: "Use Reagent Booster",
       completed: () =>
-        (!have($item`grapefruit`) && !have($item`ointment of the occult`)) ||
-        have($effect`Mystically Oiled`),
+        (!have(reagentBoosterIngredient) && !have(reagentBoosterItem)) ||
+        have(reagentBoosterEffect),
       do: (): void => {
-        if (!have($item`ointment of the occult`)) {
+        print(
+          `Completed condition: ${!have(reagentBoosterIngredient) && !have(reagentBoosterItem)}`
+        );
+        if (!have(reagentBoosterItem)) {
           if (get("reagentSummons") === 0) useSkill($skill`Advanced Saucecrafting`, 1);
-          create($item`ointment of the occult`, 1);
+          create(reagentBoosterItem, 1);
         }
-        ensureEffect($effect`Mystically Oiled`);
+        ensureEffect(reagentBoosterEffect);
       },
```

Comment:
As a non-sauceror, we are only getting a single potion from crafting. Is this sufficient for a low shiny non-sauceror?

Reply:
As a non-sauceror, this will not be sufficient if the runner has insufficient sources of other mainstat boost. It will not run as a non-Sauceror with the minimum requirements.